From: SeongJae Park <sj@kernel.org>
Subject: Docs/admin-guide/mm/damon/usage: fix a typo on the auto-tuning design reference link
Date: Wed, 21 Feb 2024 09:08:52 -0800

There was a typo on the quota auto-tuning design reference link that
caused below document build warning.  Fix it.

    WARNING: undefined label: 'damon_design_damos_quota_auto_tuning'

Link: https://lkml.kernel.org/r/20240221170852.55529-3-sj@kernel.org
Fixes: d50e871bd78b ("Docs/admin-guide/mm/damon/usage: document quota goal metric file")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lkml.kernel.org/r/20240221162745.4332955c@canb.auug.org.au
Signed-off-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/admin-guide/mm/damon/usage.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/admin-guide/mm/damon/usage.rst~docs-admin-guide-mm-damon-usage-document-quota-goal-metric-file-fix
+++ a/Documentation/admin-guide/mm/damon/usage.rst
@@ -369,7 +369,7 @@ Among the multiple feedback, the best on
 Each goal directory contains three files, namely ``target_metric``,
 ``target_value`` and ``current_value``.  Users can set and get the three
 parameters for the quota auto-tuning goals that specified on the :ref:`design
-doc <damon_design_damos_quota_auto_tuning>` by writing to and reading from each
+doc <damon_design_damos_quotas_auto_tuning>` by writing to and reading from each
 of the files.  Note that users should further write
 ``commit_schemes_quota_goals`` to the ``state`` file of the :ref:`kdamond
 directory <sysfs_kdamond>` to pass the feedback to DAMON.
_
